src/HttpKernel/RectorCIKernel.php

```diff
@@ -1,48 +1,59 @@
 
 class RectorCIKernel extends BaseKernel
 {
     use MicroKernelTrait;
 
+    /**
+     * @var string
+     */
     private const CONFIG_EXTS = '.{php,xml,yaml,yml}';
 
+    /**
+     * @return BundleInterface[]
+     */
     public function registerBundles(): iterable
     {
-        $contents = require $this->getProjectDir().'/config/bundles.php';
+        $contents = require $this->getProjectDir() . '/config/bundles.php';
         foreach ($contents as $class => $envs) {
             if ($envs[$this->environment] ?? $envs['all'] ?? false) {
                 yield new $class();
             }
         }
     }
 
-    protected function configureContainer(ContainerBuilder $container, LoaderInterface $loader): void
+    protected function configureContainer(ContainerBuilder $containerBuilder, LoaderInterface $loader): void
     {
-        $container->addResource(new FileResource($this->getProjectDir().'/config/bundles.php'));
-        $container->setParameter('container.dumper.inline_class_loader', true);
-        $confDir = $this->getProjectDir().'/config';
-
-        $loader->load($confDir.'/{packages}/*'.self::CONFIG_EXTS, 'glob');
-        $loader->load($confDir.'/{packages}/'.$this->environment.'/**/*'.self::CONFIG_EXTS, 'glob');
-        $loader->load($confDir.'/{services}'.self::CONFIG_EXTS, 'glob');
-        $loader->load($confDir.'/{services}_'.$this->environment.self::CONFIG_EXTS, 'glob');
+        $containerBuilder->addResource(new FileResource($this->getProjectDir() . '/config/bundles.php'));
+        $containerBuilder->setParameter('container.dumper.inline_class_loader', true);
+        $confDir = $this->getProjectDir() . '/config';
+
+        $loader->load($confDir . '/{packages}/*' . self::CONFIG_EXTS, 'glob');
+        $loader->load($confDir . '/{packages}/' . $this->environment . '/**/*' . self::CONFIG_EXTS, 'glob');
+        $loader->load($confDir . '/{services}' . self::CONFIG_EXTS, 'glob');
+        $loader->load($confDir . '/{services}_' . $this->environment . self::CONFIG_EXTS, 'glob');
     }
 
-    protected function configureRoutes(RouteCollectionBuilder $routes): void
+    protected function configureRoutes(RouteCollectionBuilder $routeCollectionBuilder): void
     {
-        $confDir = $this->getProjectDir().'/config';
-
-        $routes->import($confDir.'/{routes}/'.$this->environment.'/**/*'.self::CONFIG_EXTS, '/', 'glob');
-        $routes->import($confDir.'/{routes}/*'.self::CONFIG_EXTS, '/', 'glob');
-        $routes->import($confDir.'/{routes}'.self::CONFIG_EXTS, '/', 'glob');
+        $confDir = $this->getProjectDir() . '/config';
+
+        $routeCollectionBuilder->import(
+            $confDir . '/{routes}/' . $this->environment . '/**/*' . self::CONFIG_EXTS,
+            '/',
+            'glob'
+        );
+        $routeCollectionBuilder->import($confDir . '/{routes}/*' . self::CONFIG_EXTS, '/', 'glob');
+        $routeCollectionBuilder->import($confDir . '/{routes}' . self::CONFIG_EXTS, '/', 'glob');
```

Comment:
I hate this Kernel spam :angry: 

https://github.com/rectorphp/rector-ci/issues/27

Reply:
Yeah 😄 i had just a quick start without proper project set up 😄

Reply:
I know, it's not code review comment to your code.

It's just to release my frustration for this design decision in Symfony :D

Will fix in next PR

Reply:
PRs are always welcomed, i would do it otherwise myself, but it was lower on priority list 😸

Reply:
When someone appologize, I always remember this: https://www.boredpanda.com/stop-saying-sorry-say-thank-you-comic-yao-xiao :dog: 

Can I merge this?

Reply:
Thank you ❤️ 

I will review right now